jax/_src/numpy/lax_numpy.py

```diff
@@ -2469,6 +2469,17 @@ def _pad_empty(array, pad_width):
   return array
 
 
+def _pad_func(array, pad_width, func, **kwargs):
+  # Note: In numpy, `func` should modify a rank 1 array in-place.
+  # However, since Jax arrays are immutable, `func` should return
+  # the modified array.
```

Comment:
The explanation of this difference should be added to the `jnp.pad()` docstring via the `lax_description` argument; search for `lax_description` in this file to see examples.

Reply:
Thanks for the review :)
I added `lax_description` and removed the comment above.

Reply:
Thanks!